mm2src/rpc/get_raw_transaction.rs

```diff
@@ -0,0 +1,25 @@
+use crate::common::Future01CompatExt;
+use coins::{lp_coinfind, GetRawTransactionError};
+use common::mm_ctx::MmArc;
+use common::mm_error::MmError;
+use serde::{Deserialize, Serialize};
+use serde_json::Value as Json;
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub struct RawTransactionResponse {
+    pub tx_hex: String,
+}
+
+pub async fn get_raw_transaction(
+    ctx: MmArc,
+    req: Json,
```

Comment:
Could you please add `GetRawTransactionRequest` instead? Dispatcher v2 supports any request that implements/derives `Serialize`.